### ISSUE TITLE
calcite-action ARIA attributes

### ISSUE THREAD

**Issue by vivzhang** [Esri/calcite-design-system#10411]
### Check existing issues

- [X] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates and reviewed the [Accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) for guidance.

### Summary

Remove the `aria-disabled` and `aria-busy` attributes on `calcite-action`. Please ignore the request for `aria-label`, the Hub team will address that by passing in a unique value for the `label` property for their use case. 

Original AMP issues:
[11242](https://zentopia.esri.com/workspaces/sprint-board-60340b9721bc3a4ab6367db5/issues/gh/dc/hub/11242)
[11244](https://zentopia.esri.com/workspaces/sprint-board-60340b9721bc3a4ab6367db5/issues/gh/dc/hub/11244)

### Actual Behavior

`aria-disabled` and an `aria-busy` attributes rendered on the `button` inside `calcite-action` 

### Expected Behavior

No `aria-disabled` and `aria-busy` rendered on the `button` inside `calcite-action` 

### Reproduction Sample

https://codepen.io/vivzhang/pen/rNXVWBo

### Reproduction Steps

1. Open the devtool console
2. Inspect the `calcite-action` component
3. You should see the `aria-disabled` and an `aria-busy` attributes rendered on the `button` element

### Reproduction Version

2.12.1

### Priority impact

impact - p3 - not time sensitive

### Calcite package

- [ ] @esri/calcite-components
- [ ] @esri/calcite-components-angular
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

N/A

**Comment by jcfranco** [comment]
Some notes:

* `aria-disabled` – this should be addressed in today's release via https://github.com/Esri/calcite-design-system/pull/10367
* `aria-busy` – this was added here https://github.com/Esri/calcite-app-components/issues/357

**Comment by driskull** [comment]
I think we need to research more why aria-busy on an action is a problem. Would we need another property added to action to allow aria-busy to be set? like a `loadable` property?

I would expect this would apply to other components where aria-busy is being used.

**Comment by driskull** [comment]
@geospatialem can you comment on the following:

> Would we need another property added to action to allow aria-busy to be set? like a loadable property?

Is `aria-busy="false"` a problem if a button will never have it set to true? Basically, a button that doesn't need to set loading to true ever.

**Comment by geospatialem** [comment]
> Is `aria-busy="false"` a problem if a button will never have it set to true? Basically, a button that doesn't need to set loading to true ever.

@driskull Overall this is not the intent of the `aria-busy` attribute, since the state change would not be provided to assistive technologies. However, in this case where the state would not change, it might get "flagged" in accessibility tools, but it doesn't appear to effect the behavior and context we are seeking.

**Comment by driskull** [comment]
Ok, I think we can ignore `aria-busy` for now. There are other components that use `aria-busy` as well and they don't have an additional property to enable being `loading` or not.

**Comment by isaacbraun** [comment]
Verified on `4.0.0-next.24` via https://codepen.io/isaacbraun/pen/LEGLBaJ.

**Comment by nick-romano** [comment]
@geospatialem @driskull - we are still seeing the aria-busy property on actions in arcade-editor.

I'm having trouble grokking the consensus on this issue, was it deemed that aria-busy is OK?

**Comment by isaacbraun** [comment]
Hi @nick-romano 👋 Yes, it was deemed OK to keep `aria-busy` as it is still useful and doesn't hinder the experience users receive from assistive technology, even if it causes accessibility scanners to throws warnings.

Apologies that the acceptance criteria was not updated properly.

cc @geospatialem
